components/library/SDL2_image/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2024 Till Wegmueller
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		SDL2_image
+COMPONENT_VERSION=	2.8.2
+COMPONENT_SUMMARY=	Simple DirectMedia Layer - Sample Image Loading Library
+COMPONENT_PROJECT_URL=	https://libsdl.org/
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH= sha256:8f486bbfbcf8464dd58c9e5d93394ab0255ce68b51c5a966a918244820a76ddc
+COMPONENT_ARCHIVE_URL=  https://github.com/libsdl-org/SDL_image/releases/download/release-$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_FMRI=	image/library/sdl2-image
+COMPONENT_CLASSIFICATION=	System/Multimedia Libraries
+COMPONENT_LICENSE=	zlib
+COMPONENT_LICENSE_FILE=	LICENSE.txt
+
+TEST_TARGET= $(NO_TESTS)
+include $(WS_MAKE_RULES)/common.mk
+
+PATH= $(PATH.gnu)
+
+CONFIGURE_OPTIONS += --sysconfdir=/etc
+CONFIGURE_OPTIONS += --libexecdir=$(CONFIGURE_LIBDIR.$(BITS))
+CONFIGURE_OPTIONS += --disable-png-shared
```

Comment:
Are these two first settings really needed? I can't see anything in the manifest related to them.

Reply:
No, Removed.